tests/integrations/storage/test_titan.rs

```diff
@@ -431,3 +431,59 @@ fn test_delete_files_in_range_for_titan() {
         Some((Key::from_raw(b"b"), b"b_value".to_vec())),
     );
 }
+
+#[test]
+fn test_writebatch_for_titan() {
```

Comment:
it easily misses the new API after changes in raftstore. Better just choose some basic tests under `tests/integrations/raftstore` with titan enabled

Reply:
Got it, I'll choose some one, thanks!

Reply:
> it easily misses the new API after changes in raftstore. Better just choose some basic tests under `tests/integrations/raftstore` with titan enabled

@Connor1996 I chose a few tests with `titan` enabled  include `put`, `delete`,`snap` interface. PTAL if it's enough to cover, thanks!